porting/finalize/UBports-official-support/port-support-requirements.rst

```diff
@@ -0,0 +1,102 @@
+.. _PortSupportRequirements:
+
+Port support requirements
+==================================
+
+This document enumerates the requirements that must be met in order for devices to be deemed ship ready for UBports releases.
+To be considered ready, port maintainers MUST meet the requirements presented in this document, including any documents incorporated via reference.
+
+The use of “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” is per the IETF standard defined in :RFC:`2119`.
+
+-----
+
+Attaining exceptions
+********************
+
+Process
+-------
+
+* Device or software deviations from our core product (as described below) MUST be approved by the devrel team.
+* Device or software exceptions SHOULD be made via change request to this repository.
```

Comment:
"this" repo? which one? the docs?

Reply:
Yes, the docs one. I have added links to it now to make it more clear.